src/palette.c

```diff
@@ -615,7 +615,7 @@ static u8 UpdateFastPaletteFade(void)
             if (b < b0)
                 b = b0;
 
-            gPlttBufferFaded[i] = r | (g << 5) | (b << 10);
+            gPlttBufferFaded[i] = RGB(r, g, b);
```

Comment:
One day, when gPlttBufferFaded is `PlttData[] __attribute__((packed))`, this could be
```c
gPlttBufferFaded[i] = {r, g, b};
```